.gitlab-ci.yml

```diff
@@ -2,10 +2,14 @@
 stages:
   - build
   - publish
+  - deploy
   
 variables:
   DATADOG_AGENT_WINBUILDIMAGES: v5996510-a9e6a7d
   GIT_PROFILER_REF: master
+  FORCE_TRIGGER:
+    value: "false"
+    description: "Set to true to override rules in the reliability-env pipeline (e.g. override 'only deploy master')"
```

Comment:
nit: I wonder, could we rename this to `DEPLOY_TO_REL_ENV` or something like that? The naming `FORCE_TRIGGER` is a little confusing in this context (though I realise that maps to something in the rel-env)

Reply:
Makes complete sense. Fixed in 2d3888647c4dd563c06b856a38a542cca3e42257 
And tested https://gitlab.ddbuild.io/DataDog/dd-trace-dotnet/-/pipelines/6948048 (I'm so paranoiac now with CI)